pkg/crc/machine/machine.go

```diff
@@ -31,12 +29,15 @@ import (
 	"github.com/code-ready/machine/libmachine"
 	"github.com/code-ready/machine/libmachine/host"
 	"github.com/code-ready/machine/libmachine/log"
+	"github.com/code-ready/machine/libmachine/ssh"
 	"github.com/code-ready/machine/libmachine/state"
-
-	// cluster related import
-	"github.com/code-ready/crc/pkg/crc/cluster"
 )
```

Comment:
This reordering makes up the majority of the changes, it would deserve to be done separately imo.

Reply:
The previous ordering was a mess ... not sure why `make fmt` does not complain about this as was the case with Minishift,